src/frame/js/utils/functions.js

```diff
@@ -1,3 +1,18 @@
 export function bindAll(context, ...methodNames) {
     methodNames.forEach((methodName) => context[methodName] = context[methodName].bind(context));
 }
+
+export function deepFreeze(obj) {
```

Comment:
hmm yeah leftover of something I was trying to do... `Smooch.getConversation()` is returning the conversation object from the store and I wanted to deep freeze it before returning it to prevent tampering with the messages.. I'm thinking about freezing the store state after each actions though...